stumpy/stumped.py

```diff
@@ -248,20 +253,41 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True, normalize=True,
                 T_B_subseq_isconstant_future,
                 diags_futures[i],
                 ignore_trivial,
+                k,
             )
         )
 
     results = dask_client.gather(futures)
-    profile, indices = results[0]
+    profile, indices, profile_L, indices_L, profile_R, indices_R = results[0]
+
     for i in range(1, len(hosts)):
-        P, I = results[i]
-        for col in range(P.shape[1]):  # pragma: no cover
-            cond = P[:, col] < profile[:, col]
-            profile[:, col] = np.where(cond, P[:, col], profile[:, col])
-            indices[:, col] = np.where(cond, I[:, col], indices[:, col])
-
-    out[:, 0] = profile[:, 0]
-    out[:, 1:4] = indices
+        P, I, PL, IL, PR, IR = results[i]
+        for j in range(l):
+            # Uodate profie[j]
+            for D, ind in zip(P[j], I[j]):
```

Comment:
Use `idx` instead of `ind` to be consistent with other code

Reply:
I have `idx = np.searchsorted(profile[j], D, side="right")` in line270. That is why I had to choose another name. I had trouble finding a good name, and finally I just chose `ind`. 

Another idea is to use name `arg`:

```
for idx, D in zip(...):
    # some code
    arg = np.searchsorted(profile[j], D, side="right")
    # some code
```

What do you think?

Reply:
`pos` seems reasonable